### PR TITLE
Document `Producer.sendBatch`

### LINKED ISSUE #84: Document producer#sendBatch

### PR DESCRIPTION
**Pull request by Nevon**
Fixes #84

### CODE REVIEW

**Comment by Nevon** [comment]
Prettier decided to prettify the markdown. Hope you don't mind. 😅

File: README.md [hunk 1]
```diff
@@ -233,20 +233,53 @@ await producer.send({
...
+  {
+    topic: 'topic-a',
+    messages: [{ key: 'key', value: 'hello topic-a' }],
+  },
+  {
+    topic: 'topic-b',
+    messages: [{ key: 'key', value: 'hello topic-b' }],
+  }
+]
+await producer.send({ topicMessages })
```
Comment (tulios) [inline README.md]: you mean, `sendBatch` 😄

File: README.md [hunk 2]
```diff
@@ -233,20 +233,53 @@ await producer.send({
...
+```
+
+`sendBatch` has the same signature as `send`, except `topic` and `messages` are replaced with `topicMessages`:
+
+```javascript
+await producer.send({
```
Comment (tulios) [inline README.md]: should be `sendBatch`

File: README.md [hunk 3]
```diff
@@ -233,20 +233,53 @@ await producer.send({
...
+| timeout     | The time to await a response in ms. Default value _30000_                                                                                                                          |
+| compression | Compression codec. Default value `CompressionTypes.None`                                                                                                                           |
 
 By default, the producer is configured to distribute the messages with the following logic:
 
 - If a partition is specified in the message, use it
 - If no partition is specified but a key is present choose a partition based on a hash (murmur2) of the key
 - If no partition or key is present choose a partition in a round-robin fashion
 
+### <a name="producing-messages-to-multiple-topics"></a> Producing to multiple topics
```
Comment (tulios) [inline README.md]: Add this section to the summary

File: README.md (hunk 1 above)
Comment (Nevon) [reply README.md]: You saw nothing!

File: README.md (hunk 3 above)
Comment (Nevon) [reply README.md]: 1790d55

**Pull request merged**